cosmos/ovg.json

```diff
@@ -0,0 +1,54 @@
+{
+  "chainId": "ovg",
+  "chainName": "ovg",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/ovg/chain.png",
+  "rpc": "https://rpc.chain.overgold.app",
+  "rest": "https://api.chain.overgold.app",
+  "nodeProvider": {
+        "name": "Overgold",
+        "email": "info@overgold.app",
+        "website": "https://overgold.io"
+    },
+  "walletUrlForStaking": "https://staking.overgold.app",
```

Comment:
Could you remove this line as well? We allow a URL only if it utilizes Keplr wallet for staking.

Reply:
@editaahn done